HelpSource/Tutorials/Qt/create_simple_instrument_view.schelp

```diff
@@ -0,0 +1,548 @@
+title:: Qt: Interfacing an instrument with a View
+summary:: Creating a view that interacts with a Synth, introduction to Layouts
+categories:: Tutorials>Qt
+
+section:: What is this tutorial about?
+
+This is a tutorial to get started using SuperCollider's Qt implementation to strong::create views and make them interact with a Synth::.
+
+It covers those topics:
+
+list::
+## Create a keyboard from a list of midi notes
+## Create a volume slider, and a duration slider
+## Use the layout system so widgets are arranged in a grid pattern
+::
+
+section:: Setup
+
+
+You will need to strong::boot the Server::, to strong::load the SynthDef:: we will be using in this tutorial. It is a simple sine wave with a percussive enveloppe. We can control its frequency, its volume and its duration.
+
+code::
+(
+SynthDef(\instrView, { |out = 0, amp = 0.25, freq = 440, dur = 1.0|
+	var snd, env;
+
+	env = EnvGen.kr(Env.perc(dur * 0.01, dur * 0.99), doneAction: Done.freeSelf);
+	snd = SinOsc.ar(freq, mul: amp);
+	snd = snd * env;
+	snd = snd!2;
+
+	Out.ar(out, snd);
+}).add;
+)
+::
+
+You can try the synth using:
+
+code::
+Synth(\instrView)
+::
+
+We'll start with the following code:
+
+code::
+(
+var window = Window();
+var layout = HLayout();
+var button = Button();
+
+button.string_("Play");
+
+window.layout_(layout);
+layout.add(button);
+
+window.front;
+)
+::
+
+If this code does not make sense to you, you can find more explanations about it in link::Tutorials/Qt/create_window::.
+
+section:: Connecting the button
+
+So we start with a simple button, that, sadly, is not very useful so far.
+To connect it, you need to strong::assign a function to its 'action' method::.
+
+code::
+(
+var window = Window();
+var layout = HLayout();
+var button = Button();
+button.string_("Play");
+
+button.action_(
+	// The function that the button
+	// will trigger begins here...
+	{
+		Synth(\instrView);
+		"ting".postln;
+	}
+	// ...and ends here
+);
```

Comment:
I think it would be better to avoid space/newline/comment between `(` and `{`
```
button.action_({
	// This is the body of the function that the button will trigger
	Synth(\instrView);
	"ting".postln;
});
```